test/gleam/float_test.gleam

```diff
@@ -371,6 +371,29 @@ pub fn random_test() {
   |> should.be_true
 }
 
+pub fn modulo_test() {
+  float.modulo(3.0, 2.0)
+  |> should.equal(Ok(1.0))
+
+  float.modulo(1.0, 0.0)
+  |> should.equal(Error(Nil))
+
+  float.modulo(10.0, -1.0)
+  |> should.equal(Ok(0.0))
+
+  float.modulo(13.0, by: 3.0)
+  |> should.equal(Ok(1.0))
+
+  float.modulo(-13.0, by: 3.0)
+  |> should.equal(Ok(2.0))
+
+  float.modulo(13.0, by: -3.0)
+  |> should.equal(Ok(-2.0))
+
+  float.modulo(-13.0, by: -3.0)
+  |> should.equal(Ok(-1.0))
```

Comment:
All of these are whole numbers. Let's have tests for other numbers too.

Reply:
Done, could you help me solve this failing test?

```
Failures:

  1) gleam/float_test.modulo_test
     Values were not equal
     expected: Ok(1.3)
          got: Ok(1.3000000000000007)
 ```

Reply:
Check https://hexdocs.pm/gleam_stdlib/gleam/float.html#loosely_equals maybe

Reply:
> Check https://hexdocs.pm/gleam_stdlib/gleam/float.html#loosely_equals maybe

But then I believe I would need to import the `result` library to unwrap the value before, right? Is it ok adding this dependency?